Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -692,22 +695,56 @@ internal nfloat GetEstimatedRowHeight(UITableView table)
 				return CalculateHeightForCell(table, firstCell);
 			}
 
-			public override nfloat EstimatedHeight(UITableView tableView, NSIndexPath indexPath)
```

Comment:
Hopefully, by caching prototypical cells, we can compute heights fast enough so we don't need to async generate actual/estimated heights. Seems to work on the test case.